sdk/eventhub/event-hubs/src/receiver.ts

```diff
@@ -103,6 +104,10 @@ export class Receiver {
    */
   receive(onMessage: OnMessage, onError: OnError, cancellationToken?: Aborter): ReceiveHandler {
     this._throwIfReceiverOrConnectionClosed();
+    this._throwIfAlreadyReceiving();
+    if (this.getCheckpoint()) {
```

Comment:
`this.getCheckpoint()` gets called twice inside `receive()`, can we use a variable instead?